components/query_filter/utils.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_QUERY_FILTER_UTILS_H_
+#define BRAVE_COMPONENTS_QUERY_FILTER_UTILS_H_
+
+#include <string>
+
+#include "third_party/abseil-cpp/absl/types/optional.h"
+#include "url/gurl.h"
+
+namespace query_filter {
+absl::optional<GURL> ApplyQueryFilter(const GURL& original_url);
+
+/**
+ * This function will return a new url stripping any tracking query params.
+ * If nothing is to be stripped, a null value is returned.
+ *
+ * @param initiator_url Indicates the origin initiating the resource request.
```

Comment:
chromium code doesn't use `@param`. Here's an example of the normal format 
```
// Allows the embedder to programmatically control whether Site Isolation
  // should be disabled.  Note that this takes precedence over
  // ShouldEnableStrictSiteIsolation() if both return true.
  // `site_isolation_mode` specifies the site isolation mode to check; this
  // allows strict site isolation and partial site isolation to be disabled
  // according to different policies (e.g., different memory thresholds) on
  // Android.
  //
  // Note that for correctness, the same value should be consistently returned.
  virtual bool ShouldDisableSiteIsolation(
      SiteIsolationMode site_isolation_mode);
```

Reply:
if there was more than one param, they would just be listed in order using same format